icu4c/source/common/putil.cpp

```diff
@@ -1703,73 +1703,25 @@ The leftmost codepage (.xxx) wins.
 
     return posixID;
 
-#elif U_PLATFORM_USES_ONLY_WIN32_API
+#elif U_PLATFORM_HAS_WIN32_API
 #define POSIX_LOCALE_CAPACITY 64
     UErrorCode status = U_ZERO_ERROR;
-    char *correctedPOSIXLocale = 0;
+    char *correctedPOSIXLocale = nullptr;
```

Comment:
The rest of the code uses NULL, just a few lines below. Seems like this should be consistent one way or another.

Reply:
Thanks for the notice. They are fixed.